### PR TITLE
Update new Aviatrix affiliations

### PR DESCRIPTION
**Pull request by therealmitchconnors**
cc @adiprerepa, @piotrsikora, @nmittler

### CODE REVIEW

File: developers_affiliations1.txt [hunk 1]
```diff
@@ -16238,7 +16238,8 @@ PiotrProkop: PiotrProkop!users.noreply.github.com, estetus!gmail.com, piotr.prok
 	Intel Corporation
 PiotrSikora: PiotrSikora!users.noreply.github.com, piotr!cloudflare.com, piotrsikora!google.com
 	Cloudflare Inc until 2015-07-01
-	Google LLC from 2015-07-01
+	Google LLC from 2015-07-01 until 2022-08-05
+	Aviatrix from 2022-08-05
```
Comment (PiotrSikora) [inline developers_affiliations1.txt]: Aviatrix from 2022-08-08, there was a weekend in between jobs ;)

File: developers_affiliations1.txt [hunk 2]
```diff
@@ -16238,7 +16238,8 @@ PiotrProkop: PiotrProkop!users.noreply.github.com, estetus!gmail.com, piotr.prok
 	Intel Corporation
 PiotrSikora: PiotrSikora!users.noreply.github.com, piotr!cloudflare.com, piotrsikora!google.com
 	Cloudflare Inc until 2015-07-01
-	Google LLC from 2015-07-01
+	Google LLC from 2015-07-01 until 2022-08-05
```
Comment (PiotrSikora) [inline developers_affiliations1.txt]: While you're here, could you fix my previous dates as well?
```
	Cloudflare Inc from 2012-10-01 until 2015-07-10
	Google LLC from 2015-07-13 until 2022-08-05
```
Thanks!

**Review by PiotrSikora — COMMENTED** [review]
Oh, you should add our new emails to the list as well.

**Comment by nmittler** [comment]
LGTM ... thanks, Mitch!

**Comment by adiprerepa** [comment]
@therealmitchconnors could you update my affiliations as:

```
Independent until 2021-06-17
Tetrate from 2021-06-17 to 2022-10-17
Aviatrix from 2022-10-17
```

**Comment by therealmitchconnors** [comment]
> @therealmitchconnors could you update my affiliations as:
> 
> ```
> Independent until 2021-06-17
> Tetrate from 2021-06-17 to 2022-10-17
> Aviatrix from 2022-10-17
> ```

done.

**Review by adiprerepa — APPROVED** [review]

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Pull request merged**